### PR TITLE
kops: fix hash

### PR DESCRIPTION
**Pull request by flokli**
As seen on https://hydra.nixos.org/build/118009308

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/kops/default.nix [hunk 1]
```diff
@@ -58,6 +58,6 @@ in rec {
 
   kops_1_16 = mkKops {
     version = "1.16.1";
-    sha256 = "08vy57ln0qar961sf9vbrh29f04qj5siqsfim1kqwvj5xrvwa39i";
+    sha256 = "04ji1703z6b3rd4wmccah68api0jj2bndzkgdzqi855gccqbmi5g";
```
Comment (Mic92) [inline pkgs/applications/networking/cluster/kops/default.nix:61]: I am confused. Ofborg disagrees:

```
  wanted: sha256:04ji1703z6b3rd4wmccah68api0jj2bndzkgdzqi855gccqbmi5g
  got:    sha256:08vy57ln0qar961sf9vbrh29f04qj5siqsfim1kqwvj5xrvwa39i
```

Comment (Mic92) [reply pkgs/applications/networking/cluster/kops/default.nix:61]: It builds locally though?

Comment (flokli) [reply pkgs/applications/networking/cluster/kops/default.nix:61]: I did replace with all-zeros locally and TOFU-copied in the hash I got locally. Seems this is also [what hydra sees](https://hydra.nixos.org/build/117730286/nixlog/1), so I'd assume ofborg is wrong here.

Can you checkout locally too?

Comment (Mic92) [reply pkgs/applications/networking/cluster/kops/default.nix:61]: It built locally as I said.

**Pull request merged**